server_admin/topics/user-federation/ldap.adoc

```diff
@@ -1,197 +1,152 @@
+
+==== Connecting to LDAP over SSL
 
-==== Connect to LDAP over SSL
+When you configure a secure connection URL to your LDAP store (for example,`ldaps://myhost.com:636'), {project_name} uses SSL to communicate with the LDAP server. You must configure a truststore on the {project_name} server side. If you do not, {project_name} cannot trust the SSL connection to LDAP.
 
-When you configure a secured connection URL to your LDAP store (for example,`ldaps://myhost.com:636'), {project_name} will use SSL for communication with the LDAP server.
-The important thing is to properly configure a truststore on the {project_name} server side, otherwise {project_name} can't trust the SSL connection to LDAP.
+Configure the global truststore for {project_name} with the Truststore SPI. For more information about configuring the global truststore, see the link:{installguide_link}[{installguide_name}]. If you do not configure the Truststore SPI, the truststore falls back to the default mechanism provided by Java, which can be the file supplied by the `javax.net.ssl.trustStore` system property or the cacerts file from the JDK if the system property is unset.
 
-The global truststore for the {project_name} can be configured with the Truststore SPI.  Please check out the link:{installguide_link}[{installguide_name}] for more details.
-If you do not figure the truststore SPI, the truststore will fall back on the default mechanism provided by Java (either the file provided by system property `javax.net.ssl.trustStore`
-or the cacerts file from the JDK if the system property is not set).
+The `Use Truststore SPI` configuration property, in the LDAP federation provider configuration, controls the truststore SPI. By default, {project_name} sets the property to `Only for ldaps`, which is adequate for most deployments.  {project_name} uses the Truststore SPI if the connection URL to LDAP starts with `ldaps` only.
 
-There is a configuration property `Use Truststore SPI` in the LDAP federation provider configuration, where you can choose whether the Truststore SPI is used.
-By default, the value is `Only for ldaps`, which is fine for most deployments.  The Truststore SPI will only be used
-if the connection URL to LDAP starts with `ldaps`.
+==== Synchronizing LDAP users to {project_name}
 
-==== Sync of LDAP users to {project_name}
+If you set the *Import Users* option, the LDAP Provider handles importing LDAP users into the {project_name} local database. The first time a user logs in, the LDAP provider imports the LDAP user into the {project_name} database and validates the LDAP password. This first time a user logs in is the only time {project_name} imports the user. If you click the *Users*  menu in the Admin Console and click the *View all users* button, you see the LDAP users authenticated once by {project_name} only. {project_name} imports users this way, so this operation does not trigger an import of the entire LDAP user database.
```

Comment:
Not certain what the original actually means, but I think we need to move "only" from the end of the sentence to the "you will only see.." or make it "you will see only..."
However, if you understand it better, leave it as is.